openoa/schema/metadata.py

```diff
@@ -3,12 +3,15 @@
 import json
 import itertools
 from copy import deepcopy
+from typing import Any
 from pathlib import Path
 
 import yaml
 import attrs
 import numpy as np
+import pandas as pd
 from attrs import field, define
+from tabulate import tabulate
```

Comment:
Any reason to use an external library as opposed to Panda's own rendering? I believe they support ASCII rendering as `print(df)` and also support rich HTML rendering in Jupyter using `display(df).`

- [ ] I think using Panda's built in rendering is preferable, as we'd reduce the number of OpenOA dependencies.
- [ ] It would be slick, but unnecessary, to detect which environment (Jupyter notebook vs Terminal) and render HTML if possible.

Reply:
Thanks for the comments, I didn't include the full listing because it was so long, but there's no reason it couldn't be shown other than it being a snippet to highlight the change in perspective. I think it'd be easy enough to list the assets (I assume by type), so that could be included. I like the idea of a really a basic plot, but that may be difficult to get a consistent render between the terminal and a Jupyter notebook.

I fully intend to integrate it into the intro notebook once the form is more or less final.

As for tabulate, this is already a Pandas dependency for some of their rendering, and so we're not getting rid of it anytime soon, whichever way we ultimately go. The decision to use it directly, and in place of Pandas, is to have a consistent data view in a terminal and a jupyter notebook, something Pandas doesn't do well out of the box. I tried to also avoid the slick solutions because they're typically harder to maintain over time, but a quick search seems to suggest might be as simple as `is_terminal = sys.stderr.isatty()` and when True, just do normal pandas printing, and if False, print the text rendering.